### PR TITLE
Limit send-to-sidechain transaction size

### PR DESCRIPTION
**Pull request by pstratem**
Fix issue #15

### CODE REVIEW

File: contrib/sidechain-manipulation.py [hunk 1]
```diff
@@ -118,7 +118,26 @@ def __init__(self, value):
...
+			tx_hex = bitcoin.createrawtransaction([], {send_address: Decimal(sys.argv[3])})
+			tx_hex = bitcoin.fundrawtransaction(tx_hex)['hex']
+			tx = bitcoin.signrawtransaction(tx_hex)
+			assert(tx['complete'])
+
+			tx_hex = tx['hex']
+
+			total_length  = len(tx_hex)/2
+			total_length += 14*32
+			total_length += 1000 # wild guess
```
Comment (TheBlueMatt) [inline contrib/sidechain-manipulation.py]: I'm assuming this was calculated by some average?

File: contrib/sidechain-manipulation.py [hunk 2]
```diff
@@ -118,7 +118,26 @@ def __init__(self, value):
...
+		try:
+			tx_hex = bitcoin.createrawtransaction([], {send_address: Decimal(sys.argv[3])})
+			tx_hex = bitcoin.fundrawtransaction(tx_hex)['hex']
+			tx = bitcoin.signrawtransaction(tx_hex)
+			assert(tx['complete'])
+
+			tx_hex = tx['hex']
+
+			total_length  = len(tx_hex)/2
+			total_length += 14*32
```
Comment (TheBlueMatt) [inline contrib/sidechain-manipulation.py]: Is this the max merkle tree size or so? Can you add a comment for that ?

File: contrib/sidechain-manipulation.py (hunk 1 above)
Comment (pstratem) [inline contrib/sidechain-manipulation.py]: this was calculated by adding together full_contract, scriptSecond
then i added an entirely arbitrary amount to that

File: contrib/sidechain-manipulation.py (hunk 2 above)
Comment (pstratem) [inline contrib/sidechain-manipulation.py]: yes and yes

**Comment by sipa** [comment]
Untested ACK

**Comment by pstratem** [comment]
@TheBlueMatt please review kthx

**Pull request merged**